runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -115,13 +110,38 @@ protected void shutDown() throws Exception
 	@Subscribe
 	public void onWidgetLoaded(WidgetLoaded event)
 	{
-		if (event.getGroupId() == WidgetID.BA_REWARD_GROUP_ID)
```

Comment:
Why have you removed the stuff that handles the Wave Times option?

Reply:
I haven't, I just moved it into the switch statement.

Reply:
Ah, that's a confusing diff. Nvm then